resonances/config_fit_Ds2starplus.yml

```
@@ -0,0 +1,11 @@
+input: outputs/Ds2starplus/Dplus_K0S_HM_centsel.parquet.gzip
+output_dir: outputs/Ds2starplus
+pdg_reso: 435
+
+fit_config:
+  pt_mins: [2.]
+  pt_maxs: [20.]
+  mass_mins: [0.55, 0.55]
+  mass_maxs: [0.85, 0.85]
```

```suggestion
  mass_mins: [0.55]
  mass_maxs: [0.85]
```
This looks like a remnant of a multi-bin fit.